android/assets/jsons/translations/template.properties

```diff
@@ -360,6 +360,8 @@ Three Continents =
 Four Corners = 
 Archipelago = 
 Inner Sea = 
+Random number of Players =
+Random number of City-States =
```

Comment:
tests fail because you didn't  add a space after `=`

Reply:
Thanks, fixed